docs/index.md

```diff
@@ -413,13 +413,27 @@ a different working order is required.
   See for example "`The book of bobbin lace stitches`" by Bridget M. Cook and Geraldine Stott. 
 
 
-### <a name="H18-variants"/>H18 &mdash; two patterns
+### <a name="h18variations"/>H18 &mdash; two patterns
 
 [![](w/page241a.gif)][P241]      
 
 The work consists of two parts, starting with [Ground a][H18a], and changing halfway to [Ground b][H18b].   
 
+## Three&dash;paired joins   
 
+Groundforge can not do real three&dash;paired joins. However, we can simulate such a join by treating it as a series of stitches. See [MAE-GF][p-mae-gf-3] for some examples.      
+### <a name="a13joins"/>A13 &mdash; examples of three&dash;paired joins
```

Comment:
I would expect this one in the side bar too. Subsequent sections don't follow the proper hyrarchy in the sidebar.

I created a sneak preview for you on https://jo-pol.github.io/gw-lace-to-gf/
You can paste the content of the sidebar in a local html file for some testing.

Reply:
I changed the side-bar (and corrected H18). Its in the first set of commits.
Will have to change it again as the title of the paragraph changes.

Reply:
Forgot a ' " '....

Reply:
Notice: I can test the side bar by copying the table in my test.md. 
The preview shows the layout.